QualificationsRegister/Functions/Public/OrganisationsPublic.cs

```diff
@@ -62,19 +67,14 @@ public async Task<HttpResponseData> GetOrganisationsList([HttpTrigger(Authorizat
         /// <param name="number"></param>
         /// <returns></returns>
         [Function("Organisation")]
-        public async Task<HttpResponseData> GetOrganisation([HttpTrigger(AuthorizationLevel.Function, "get", "{number:string?}")] HttpRequestData req, string? number)
+        public async Task<HttpResponseData> GetOrganisation([HttpTrigger(AuthorizationLevel.Function, "get",
+            Route = "organisation/{number?}")] HttpRequestData req, string? number)
```

Comment:
Pls update the route to _Route = "organisations/{number?}_ .

Note the plural organisation**s** instead of singular organisation. Something requested by Toby.

Reply:
fixed